README.md

```diff
@@ -25,7 +25,7 @@ document.querySelector('pre > code').innerHTML = codeHTML
 
 ### Highlight with CSS
 
-Then make your own theme with customized colors by token type and put in global CSS. The corresponding class names star with `sh__` prefix.
+Then make your own theme with customized colors by token type and put in global CSS. The corresponding class names starts with `--sh-` prefix.
```

Comment:
Seem we need to use plural here for "class names"
```suggestion
Then make your own theme with customized colors by token type and put in global CSS. The corresponding class names start with `--sh-` prefix.
```

Reply:
you're right, my bad